blocks/api/test/registration.js

```diff
@@ -73,7 +73,18 @@ describe( 'blocks', () => {
 		it( 'should accept valid block names', () => {
 			const block = registerBlockType( 'my-plugin/fancy-block-4', defaultBlockSettings );
 			expect( console.error ).not.toHaveBeenCalled();
-			expect( block ).toEqual( { name: 'my-plugin/fancy-block-4', icon: 'block-default', save: noop, category: 'common', title: 'block title' } );
+			expect( block ).toEqual( {
+				name: 'my-plugin/fancy-block-4',
+				icon: 'block-default',
+				save: noop,
+				category: 'common',
+				title: 'block title',
+				attributes: {
+					className: {
```

Comment:
I wasn't aware it was possible, this is a nice way to ensuring that `className` is defined as a string 👍